java/ql/src/experimental/Security/CWE/CWE-555/PasswordInConfigurationFile.ql

```diff
@@ -0,0 +1,49 @@
+/**
+ * @name Password in configuration file
+ * @description Finds passwords in configuration files.
+ * @kind problem
+ * @id java/password-in-configuration
+ * @tags security
+ *       external/cwe/cwe-555
+ *       external/cwe/cwe-256
+ *       external/cwe/cwe-260
+ */
+
+import java
+
+/* Holds if the attribute value is not a cleartext password */
```

Comment:
Regular comments should be `/**` doc comments (here and below)

Reply:
Thanks @smowton. I've made the change.